### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by alryzden**

### CODE REVIEW

File: Task2/pkg/builder/car.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	c.accelerator = true
+}
+
+// AddGun builds a gun.
+func (c *car) AddGun() {
+	c.gun = true
+}
+
+// MakeDescription create description car.
+func (c *car) MakeDescription() {
```
Comment (Aber4Nod) [inline Task2/pkg/builder/car.go]: GetDescription и возвращай описание

Comment (alryzden) [reply Task2/pkg/builder/car.go]: Done

File: Task2/cmd/builder/main.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"github.com/RyzhAlexWork/gointern/Task2/pkg/builder"
+)
+
+func main() {
+	newCar := builder.NewCar()
+	build := builder.NewBuild(newCar)
```
Comment (starosta46) [inline Task2/cmd/builder/main.go]: carBuilder

Comment (alryzden) [reply Task2/cmd/builder/main.go]: Done

**Pull request merged**